brayns/engine/core/Engine.cpp

```diff
@@ -120,11 +104,10 @@ Engine::OsprayModuleHandler::~OsprayModuleHandler()
 Engine::Engine(ParametersManager &parameters)
     : _params(parameters)
     , _osprayDevice(OsprayDeviceInitializer::init(parameters))
-    , _camera(std::make_unique<PerspectiveCamera>())
-    , _renderer(std::make_unique<InteractiveRenderer>())
+    , _camera(Perspective())
+    , _renderer(Interactive())
```

Comment:
I would still call that PerspectiveProjection and InteractiveRenderer but if it is too long it's ok like that also.

Reply:
I prefer this naming. Also for the renderer would be weird as the containing class is already called Renderer.